test/e2e/framework/util.go

```diff
@@ -1406,7 +1408,9 @@ retriesLoop:
 			fmt.Println("invariants violated:\n", strings.Join(errs.List(), "\n - "))
 			continue retriesLoop
 		}
-		ExpectEqual(errs.Len() > 0, false, strings.Join(errs.List(), "\n - "))
+		if errs.Len() > 0 {
+			Failf("Error: %v", strings.Join(errs.List(), "\n - "))
```

Comment:
```suggestion
			Failf("Unexpected error(s): %v", strings.Join(errs.List(), "\n - "))
```

Reply:
gomega.Expect will be used don't you think